### PR TITLE
fix(Offline): add no index in offline page

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add no index in offline page and in new features to search robots no index this page.

### CODE REVIEW

**Comment by fcoronelmakingsense** [comment]
test this please

File: src/components/Offline/Offline.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 import React from 'react';
+import { Helmet } from 'react-helmet';
 
 const Offline = () => (
   <div className="dp-app-container">
+    <Helmet>
+      <meta name="robots" content="noindex,nofollow" />
+    </Helmet>
```
Comment (andresmoschini) [inline src/components/Offline/Offline.js:8]: It should be also added in `NewFeatures.js`, right?

Comment (fcoronelmakingsense) [reply src/components/Offline/Offline.js:8]: you right, good catch!

**Comment by fcoronelmakingsense** [comment]
test this please

File: src/components/Offline/Offline.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Offline/Offline.js:8]: changed

File: src/components/Offline/Offline.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import React from 'react';
+import { Helmet } from 'react-helmet';
 
 const Offline = () => (
   <div className="dp-app-container">
+    <Helmet>
+      <meta name="robots" content="noindex,nofollow" />
```
Comment (cbernat) [inline src/components/Offline/Offline.js:7]: ❤️

**Pull request merged**